app/serializers/application_serializer.rb

```diff
@@ -3,6 +3,7 @@ class ApplicationSerializer < ActiveModel::Serializer
   delegate :to_html, to: :string_formatter
   delegate :to_unit, to: :number_formatter
   delegate :to_delimited, to: :number_formatter
+  delegate :distance_of_time_in_words_from_now, to: :date_formatter
```

Comment:
Delegate the message to `distance_of_time_in_words_from_now` to the method `date_formatter`.